src/module.ml

```diff
@@ -24,6 +24,10 @@ module Name = struct
   let pp = Format.pp_print_string
   let pp_quote fmt x = Format.fprintf fmt "%S" x
 
+  let stdlib = "Stdlib"
+
+  let is_caml_internal = String.is_prefix ~prefix:"Camlinternal"
```

Comment:
It seems a bit strange to hardcode these names here. For example, this PR also adds `(modules_before_stdlib)` which I assume is a constant for the ocaml compiler as well. Can these be configured or somehow detected?

Reply:
I suppose yes, that will make it easier to change these parameters in the compiler

Reply:
Done